packages/components/src/Layout/Layout.scss

```diff
@@ -5,7 +5,7 @@
 
 /// Layout height of the header
 /// @type Number (Unit)
-$tc-drawer-z-index: $drawer-z-index !default;
+$tc-drawer-z-index: $zindex-navbar-fixed + 6 !default;
```

Comment:
Did we consider closing the typeahead menu while clicking its CTA instead?

Reply:
At least, adding a comment on top of that magic number could be nice!

Reply:
Good point, just add it